cli/cmd/upgrade_project_test.go

```diff
@@ -155,7 +155,7 @@ func Test_transformShipyard(t *testing.T) {
 									Name: "rollback",
 									TriggeredOn: []keptnv2.Trigger{
```

Comment:
I think we need to check wether the `deployment` task in the `delivery` sequence uses a `direct` or `blue_green_service` deployment strategy, and only add the `rollback` sequence if the `blue_green_service` strategy is being used

Reply:
well, this absolutely makes sense, sir :)